src/main/scala/net/snowflake/spark/snowflake/SnowflakeRelation.scala

```diff
@@ -193,7 +193,7 @@ private[snowflake] case class SnowflakeRelation(
   ): RDD[T] = {
     val conn = DefaultJDBCWrapper.getConnector(params)
     try {
-      Utils.genPrologueSql(params).foreach(x => x.execute(bindVariableEnabled = false)(conn))
+      Utils.genPrologueSql(params).execute(bindVariableEnabled = false)(conn)
```

Comment:
We can also avoid merging timezone setting query and session parameter setting query into one query. But it would cost us some performances, so I wanted to merge them into one query.